dist/index.js

```diff
@@ -27206,41 +27206,46 @@ utils.intFromLE = intFromLE;
 
 },{"bn.js":19,"minimalistic-assert":77,"minimalistic-crypto-utils":78}],48:[function(require,module,exports){
 module.exports={
-  "_from": "elliptic@^6.2.3",
+  "_args": [
+    [
+      "elliptic@6.4.1",
+      "/Users/gavinatkinson/Tlon/urbit-key-generation"
```

Comment:
Local paths here too! But I see local paths in the old version too. I guess that's safe to ignore? (And won't matter if we end up gitignoring this.)

Reply:
This looks like a build artefact, something to do with browserify. It is present in the /dist/index.js now, but with Jared's paths